[{"tags": ["neural-network", "pytorch", "logistic-regression", "cross-entropy"], "owner": {"account_id": 4832003, "reputation": 933, "user_id": 3899975, "user_type": "registered", "accept_rate": 85, "profile_image": "https://www.gravatar.com/avatar/?s=256&d=identicon&r=PG&f=1", "display_name": "A.E", "link": "https://stackoverflow.com/users/3899975/a-e"}, "is_answered": true, "view_count": 1428, "accepted_answer_id": 58368072, "answer_count": 2, "score": 1, "last_activity_date": 1571019959, "creation_date": 1570934146, "last_edit_date": 1571019959, "question_id": 58360150, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/58360150/how-to-write-custom-crossentropyloss", "title": "How to write custom CrossEntropyLoss"}, {"tags": ["python", "machine-learning", "pytorch"], "owner": {"account_id": 19777029, "reputation": 31, "user_id": 14482749, "user_type": "registered", "profile_image": "https://lh5.googleusercontent.com/-gVanv1T57zI/AAAAAAAAAAI/AAAAAAAAACw/AMZuuckW465m3aCFUIiCy06RsvyKNTq-PA/s96-c/photo.jpg?sz=256", "display_name": "Curerious", "link": "https://stackoverflow.com/users/14482749/curerious"}, "is_answered": true, "view_count": 371, "answer_count": 2, "score": 1, "last_activity_date": 1603451457, "creation_date": 1603433738, "last_edit_date": 1603451457, "question_id": 64494819, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/64494819/use-crossentropyloss-with-logsoftmax", "title": "Use CrossEntropyLoss with LogSoftmax"}, {"tags": ["python", "neural-network", "deep-learning", "conv-neural-network", "pytorch"], "owner": {"account_id": 11188804, "reputation": 577, "user_id": 8210067, "user_type": "registered", "accept_rate": 56, "profile_image": "https://www.gravatar.com/avatar/60bc6f627b6ed9b5c1db1145bdae24b7?s=256&d=identicon&r=PG&f=1", "display_name": "Rajat", "link": "https://stackoverflow.com/users/8210067/rajat"}, "is_answered": true, "view_count": 16412, "accepted_answer_id": 47488703, "answer_count": 2, "score": 6, "last_activity_date": 1628213106, "creation_date": 1511630107, "question_id": 47488598, "content_license": "CC BY-SA 3.0", "link": "https://stackoverflow.com/questions/47488598/attributeerror-crossentropyloss-object-has-no-attribute-backward", "title": "AttributeError: &#39;CrossEntropyLoss&#39; object has no attribute &#39;backward&#39;"}, {"tags": ["machine-learning", "deep-learning", "pytorch", "cross-entropy"], "owner": {"account_id": 9255686, "reputation": 35, "user_id": 6874682, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/ab1aa1a747cfa3d3f1d638304ce55e2a?s=256&d=identicon&r=PG&f=1", "display_name": "Thomas231", "link": "https://stackoverflow.com/users/6874682/thomas231"}, "is_answered": true, "view_count": 358, "accepted_answer_id": 65653065, "answer_count": 1, "score": 0, "last_activity_date": 1610281379, "creation_date": 1610277943, "last_edit_date": 1610280606, "question_id": 65652731, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/65652731/crossentropyloss-pytorch-targetsize-does-not-match-torchsize", "title": "Crossentropyloss Pytorch: Targetsize does not match Torchsize"}, {"tags": ["python", "deep-learning", "pytorch"], "owner": {"account_id": 13660280, "reputation": 411, "user_id": 9855577, "user_type": "registered", "profile_image": "https://lh3.googleusercontent.com/-XdUIqdMkCWA/AAAAAAAAAAI/AAAAAAAAAAA/4252rscbv5M/photo.jpg?sz=256", "display_name": "Umair Javaid", "link": "https://stackoverflow.com/users/9855577/umair-javaid"}, "is_answered": true, "view_count": 636, "accepted_answer_id": 59395142, "answer_count": 1, "score": 0, "last_activity_date": 1577880100, "creation_date": 1576679549, "question_id": 59394244, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/59394244/pytorch-how-to-access-crossentropyloss-gradient", "title": "Pytorch: How to access CrossEntropyLoss() gradient?"}, {"tags": ["python", "numpy", "tensorflow", "pytorch"], "owner": {"account_id": 16628588, "reputation": 13, "user_id": 12017005, "user_type": "registered", "profile_image": "https://lh3.googleusercontent.com/a-/AAuE7mAVfVjWhCd_8BH5NH78oRuYx0WGwv41pvgBCj6D=k-s256", "display_name": "Nick Koutroumpinis", "link": "https://stackoverflow.com/users/12017005/nick-koutroumpinis"}, "is_answered": true, "view_count": 156, "accepted_answer_id": 65144531, "answer_count": 1, "score": 0, "last_activity_date": 1607088889, "creation_date": 1607084249, "question_id": 65143404, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/65143404/pytorch-crossentropyloss-tensorflow-equivalent", "title": "Pytorch CrossEntropyLoss Tensorflow Equivalent"}, {"tags": ["pytorch", "cross-entropy"], "owner": {"account_id": 296054, "reputation": 2570, "user_id": 600791, "user_type": "registered", "accept_rate": 100, "profile_image": "https://i.stack.imgur.com/KXUqD.jpg?s=256&g=1", "display_name": "JacKeown", "link": "https://stackoverflow.com/users/600791/jackeown"}, "is_answered": true, "view_count": 16097, "accepted_answer_id": 65193236, "answer_count": 1, "score": 16, "last_activity_date": 1607403191, "creation_date": 1607396412, "question_id": 65192475, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/65192475/pytorch-logsoftmax-vs-softmax-for-crossentropyloss", "title": "PyTorch LogSoftmax vs Softmax for CrossEntropyLoss"}, {"tags": ["python", "python-3.x", "neural-network", "pytorch", "conv-neural-network"], "owner": {"account_id": 17868857, "reputation": 11, "user_id": 12980617, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/650cff3595acc6574f78ea251c470cfb?s=256&d=identicon&r=PG&f=1", "display_name": "sv98bc", "link": "https://stackoverflow.com/users/12980617/sv98bc"}, "is_answered": true, "view_count": 1563, "accepted_answer_id": 65013065, "answer_count": 1, "score": 0, "last_activity_date": 1606340779, "creation_date": 1606339766, "last_edit_date": 1606340779, "question_id": 65012948, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/65012948/pytorch-39crossentropylossquot-object-has-no-attribute-39item39", "title": "PyTorch: &amp;#39;CrossEntropyLoss&amp;quot; object has no attribute &amp;#39;item&amp;#39;"}, {"tags": ["python", "pytorch"], "owner": {"account_id": 20456040, "reputation": 38, "user_id": 15009686, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/6ebbd270a68153928383024701f02ff7?s=256&d=identicon&r=PG&f=1", "display_name": "Bradley Thompson", "link": "https://stackoverflow.com/users/15009686/bradley-thompson"}, "is_answered": true, "view_count": 117, "accepted_answer_id": 65733864, "answer_count": 1, "score": 1, "last_activity_date": 1610704768, "creation_date": 1610679057, "question_id": 65729822, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/65729822/running-into-indexerror-when-following-pytorch-crossentropyloss-example-for-clas", "title": "Running into IndexError when following PyTorch CrossEntropyLoss example for classification"}, {"tags": ["pytorch"], "owner": {"account_id": 405323, "reputation": 2638, "user_id": 774133, "user_type": "registered", "accept_rate": 48, "profile_image": "https://www.gravatar.com/avatar/5c30305b8e03ae14c13dc86a79e111c8?s=256&d=identicon&r=PG", "display_name": "Antonio Sesto", "link": "https://stackoverflow.com/users/774133/antonio-sesto"}, "is_answered": false, "view_count": 345, "answer_count": 1, "score": 0, "last_activity_date": 1644838174, "creation_date": 1644664236, "question_id": 71091323, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/71091323/crossentropyloss-on-sequences", "title": "CrossEntropyLoss on sequences"}, {"tags": ["deep-learning", "pytorch"], "owner": {"account_id": 14247933, "reputation": 45, "user_id": 10292378, "user_type": "registered", "profile_image": "https://lh4.googleusercontent.com/-RchaD2y6Irk/AAAAAAAAAAI/AAAAAAAABQc/rNQNGuP714w/photo.jpg?sz=256", "display_name": "Marcel Coetzee", "link": "https://stackoverflow.com/users/10292378/marcel-coetzee"}, "is_answered": true, "view_count": 125, "accepted_answer_id": 69123667, "answer_count": 3, "score": 0, "last_activity_date": 1631227447, "creation_date": 1631213571, "last_edit_date": 1631215108, "question_id": 69123404, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/69123404/crossentropyloss-equivalence-to-logsoftmax-nllloss", "title": "CrossEntropyLoss equivalence to LogSoftmax + NLLLoss"}, {"tags": ["neural-network", "deep-learning", "pytorch", "tensor"], "owner": {"account_id": 453163, "reputation": 3359, "user_id": 1276374, "user_type": "registered", "accept_rate": 48, "profile_image": "https://www.gravatar.com/avatar/7ce4a877cf573491b23fc6aa1bf18a1d?s=256&d=identicon&r=PG", "display_name": "Boris Mocialov", "link": "https://stackoverflow.com/users/1276374/boris-mocialov"}, "is_answered": true, "view_count": 565, "answer_count": 1, "score": 0, "last_activity_date": 1521048152, "creation_date": 1521038798, "last_edit_date": 1521039243, "question_id": 49280731, "content_license": "CC BY-SA 3.0", "link": "https://stackoverflow.com/questions/49280731/tensor-sizes-crossentropyloss", "title": "Tensor sizes CrossEntropyLoss"}, {"tags": ["pytorch", "autograd"], "owner": {"account_id": 4842611, "reputation": 18149, "user_id": 3907250, "user_type": "registered", "accept_rate": 79, "profile_image": "https://i.stack.imgur.com/2u77o.jpg?s=256&g=1", "display_name": "aerin", "link": "https://stackoverflow.com/users/3907250/aerin"}, "is_answered": true, "view_count": 3708, "accepted_answer_id": 54166311, "answer_count": 1, "score": 6, "last_activity_date": 1547404177, "creation_date": 1547355435, "last_edit_date": 1547359018, "question_id": 54166206, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/54166206/grad-outputs-in-torch-autograd-grad-crossentropyloss", "title": "grad_outputs in torch.autograd.grad (CrossEntropyLoss)"}, {"tags": ["pytorch", "loss-function"], "owner": {"account_id": 21518512, "reputation": 131, "user_id": 15859984, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/524e12313d573b1d4965931fbf2773db?s=256&d=identicon&r=PG&f=1", "display_name": "clueless", "link": "https://stackoverflow.com/users/15859984/clueless"}, "is_answered": true, "view_count": 9723, "answer_count": 5, "score": 5, "last_activity_date": 1659705767, "creation_date": 1622151723, "last_edit_date": 1622152956, "question_id": 67730325, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/67730325/using-weights-in-crossentropyloss-and-bceloss-pytorch", "title": "Using weights in CrossEntropyLoss and BCELoss (PyTorch)"}, {"tags": ["python", "pytorch"], "owner": {"account_id": 1053202, "reputation": 54818, "user_id": 1056563, "user_type": "registered", "accept_rate": 90, "profile_image": "https://www.gravatar.com/avatar/bc888aa658ab0e0d2a4d305ad7f69123?s=256&d=identicon&r=PG", "display_name": "WestCoastProjects", "link": "https://stackoverflow.com/users/1056563/westcoastprojects"}, "is_answered": true, "view_count": 80, "accepted_answer_id": 64483050, "answer_count": 2, "score": 0, "last_activity_date": 1603377392, "creation_date": 1603338670, "question_id": 64475041, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/64475041/pytorch-crossentropyloss-from-single-dimensional-tensors", "title": "Pytorch CrossEntropyLoss from single dimensional Tensors"}, {"tags": ["python", "python-3.x", "neural-network", "pytorch"], "owner": {"account_id": 19235035, "reputation": 1026, "user_id": 14056167, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/dcf99b37dd324ca23facb48577049780?s=256&d=identicon&r=PG&f=1", "display_name": "Christian Doucette", "link": "https://stackoverflow.com/users/14056167/christian-doucette"}, "is_answered": true, "view_count": 3922, "accepted_answer_id": 65077272, "answer_count": 2, "score": 2, "last_activity_date": 1606753324, "creation_date": 1606699214, "question_id": 65066981, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/65066981/pytorch-valueerror-expected-target-size-2-13-got-torch-size2-when-calli", "title": "Pytorch ValueError: Expected target size (2, 13), got torch.Size([2]) when calling CrossEntropyLoss"}, {"tags": ["pytorch", "loss-function", "loss", "temporal"], "owner": {"account_id": 22607614, "reputation": 1, "user_id": 16788299, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/dae85e07776232461c61d4bc0e9fdf89?s=256&d=identicon&r=PG&f=1", "display_name": "papillon", "link": "https://stackoverflow.com/users/16788299/papillon"}, "is_answered": false, "view_count": 84, "answer_count": 2, "score": 0, "last_activity_date": 1657202972, "creation_date": 1657201629, "question_id": 72898992, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/72898992/crossentropyloss-for-multi-label-time-series", "title": "CrossEntropyLoss for multi-label time series"}, {"tags": ["neural-network", "pytorch", "softmax", "cross-entropy"], "owner": {"account_id": 19529, "reputation": 29441, "user_id": 45843, "user_type": "registered", "accept_rate": 91, "profile_image": "https://www.gravatar.com/avatar/55f76441107f14055f95e08bb7bd01df?s=256&d=identicon&r=PG&f=1", "display_name": "rwallace", "link": "https://stackoverflow.com/users/45843/rwallace"}, "is_answered": true, "view_count": 19, "accepted_answer_id": 73596209, "answer_count": 1, "score": 2, "last_activity_date": 1662253489, "creation_date": 1662204784, "question_id": 73591918, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/73591918/crossentropyloss-showing-poor-accuracy-on-2d-output", "title": "CrossEntropyLoss showing poor accuracy on 2d output"}, {"tags": ["python", "python-3.x", "machine-learning", "neural-network", "pytorch"], "owner": {"account_id": 13109453, "reputation": 177, "user_id": 9471025, "user_type": "registered", "profile_image": "https://graph.facebook.com/1710846528938122/picture?type=large", "display_name": "Minoh Kim", "link": "https://stackoverflow.com/users/9471025/minoh-kim"}, "is_answered": true, "view_count": 21120, "accepted_answer_id": 49209628, "answer_count": 1, "score": 15, "last_activity_date": 1520695019, "creation_date": 1520667400, "last_edit_date": 1520695019, "question_id": 49206550, "content_license": "CC BY-SA 3.0", "link": "https://stackoverflow.com/questions/49206550/pytorch-error-multi-target-not-supported-in-crossentropyloss", "title": "pytorch error: multi-target not supported in CrossEntropyLoss()"}, {"tags": ["python", "pytorch", "conv-neural-network", "google-colaboratory", "cross-entropy"], "owner": {"account_id": 14067774, "reputation": 433, "user_id": 10162123, "user_type": "registered", "profile_image": "https://i.stack.imgur.com/pr5u6.png?s=256&g=1", "display_name": "Parthik B.", "link": "https://stackoverflow.com/users/10162123/parthik-b"}, "is_answered": true, "view_count": 4307, "answer_count": 2, "score": 0, "last_activity_date": 1585758715, "creation_date": 1585697390, "last_edit_date": 1585729865, "question_id": 60961466, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/60961466/pytorch-multi-target-error-with-crossentropyloss", "title": "Pytorch: multi-target error with CrossEntropyLoss"}, {"tags": ["neural-network", "pytorch", "conv-neural-network"], "owner": {"account_id": 20901332, "reputation": 1, "user_id": 15354380, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/63a1576fd302434e1c12aba75daa83fa?s=256&d=identicon&r=PG&f=1", "display_name": "confusedsoul", "link": "https://stackoverflow.com/users/15354380/confusedsoul"}, "is_answered": false, "view_count": 378, "answer_count": 2, "score": 0, "last_activity_date": 1615288373, "creation_date": 1615217568, "question_id": 66532635, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/66532635/error-while-executing-crossentropyloss-in-pytorch", "title": "Error while executing CrossEntropyLoss() in PyTorch"}, {"tags": ["python", "neural-network", "pytorch"], "owner": {"account_id": 11196448, "reputation": 375, "user_id": 8214951, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/5a10dc667b4512ae21e736702ac054c5?s=256&d=identicon&r=PG&f=1", "display_name": "flying_loaf_3", "link": "https://stackoverflow.com/users/8214951/flying-loaf-3"}, "is_answered": false, "view_count": 31, "answer_count": 1, "score": 0, "last_activity_date": 1662082597, "creation_date": 1662079179, "question_id": 73576683, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/73576683/pytorch-crossentropyloss-dimension-out-of-range", "title": "PyTorch CrossEntropyLoss DImension Out of Range"}, {"tags": ["python", "neural-network", "pytorch"], "owner": {"account_id": 6518381, "reputation": 586, "user_id": 5043357, "user_type": "registered", "accept_rate": 36, "profile_image": "https://www.gravatar.com/avatar/efeb4b2e5899c59b9e65008393ca5019?s=256&d=identicon&r=PG&f=1", "display_name": "rishran", "link": "https://stackoverflow.com/users/5043357/rishran"}, "is_answered": true, "view_count": 338, "accepted_answer_id": 66138131, "answer_count": 1, "score": -2, "last_activity_date": 1612964835, "creation_date": 1612934767, "last_edit_date": 1612963521, "question_id": 66131350, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/66131350/why-do-i-get-a-valueerror-when-using-crossentropyloss", "title": "Why do I get a ValueError when using CrossEntropyLoss"}, {"tags": ["python", "pytorch"], "owner": {"account_id": 14064207, "reputation": 487, "user_id": 10159479, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/835dda9b64c9a803249e71b188548049?s=256&d=identicon&r=PG&f=1", "display_name": "Greg", "link": "https://stackoverflow.com/users/10159479/greg"}, "is_answered": true, "view_count": 111641, "accepted_answer_id": 52948967, "answer_count": 4, "score": 44, "last_activity_date": 1654951684, "creation_date": 1540290991, "last_edit_date": 1654951684, "question_id": 52946920, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/52946920/bool-value-of-tensor-with-more-than-one-value-is-ambiguous-in-pytorch", "title": "Bool value of Tensor with more than one value is ambiguous in Pytorch"}, {"tags": ["image-segmentation", "pytorch", "loss-function", "cross-entropy", "semantic-segmentation"], "owner": {"account_id": 11444653, "reputation": 886, "user_id": 8388965, "user_type": "registered", "profile_image": "https://i.stack.imgur.com/FSFRG.png?s=256&g=1", "display_name": "Farshid Rayhan", "link": "https://stackoverflow.com/users/8388965/farshid-rayhan"}, "is_answered": true, "view_count": 11234, "accepted_answer_id": 53370724, "answer_count": 2, "score": 4, "last_activity_date": 1552064121, "creation_date": 1529236842, "last_edit_date": 1529300503, "question_id": 50896412, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/50896412/channel-wise-crossentropyloss-for-image-segmentation-in-pytorch", "title": "Channel wise CrossEntropyLoss for image segmentation in pytorch"}, {"tags": ["pytorch"], "owner": {"account_id": 2521337, "reputation": 5786, "user_id": 2191652, "user_type": "registered", "accept_rate": 68, "profile_image": "https://www.gravatar.com/avatar/3cfa80e86236c34a19d11dfa54262d40?s=256&d=identicon&r=PG", "display_name": "mcExchange", "link": "https://stackoverflow.com/users/2191652/mcexchange"}, "is_answered": true, "view_count": 2636, "answer_count": 2, "score": 2, "last_activity_date": 1512817240, "creation_date": 1512740935, "question_id": 47715696, "content_license": "CC BY-SA 3.0", "link": "https://stackoverflow.com/questions/47715696/example-crossentropyloss-for-3d-semantic-segmentation-in-pytorch", "title": "Example CrossEntropyLoss for 3D semantic segmentation in pytorch"}, {"tags": ["python", "pytorch"], "owner": {"account_id": 13839801, "reputation": 581, "user_id": 9990438, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/f465ecc9dca813e1dfa52b0e6d77d1b1?s=256&d=identicon&r=PG&f=1", "display_name": "whtitefall", "link": "https://stackoverflow.com/users/9990438/whtitefall"}, "is_answered": true, "view_count": 118, "accepted_answer_id": 64629032, "answer_count": 1, "score": 0, "last_activity_date": 1604209534, "creation_date": 1604189604, "last_edit_date": 1604208912, "question_id": 64627576, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/64627576/pass-weights-into-crossentropyloss-in-correct-order", "title": "Pass weights into CrossEntropyLoss in correct order"}, {"tags": ["python", "tensorflow", "machine-learning", "deep-learning", "pytorch"], "owner": {"account_id": 23497849, "reputation": 27, "user_id": 17548063, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/021c6e520f1edf0afcf183054b5df3c5?s=256&d=identicon&r=PG", "display_name": "speedhawk1", "link": "https://stackoverflow.com/users/17548063/speedhawk1"}, "is_answered": true, "view_count": 243, "accepted_answer_id": 71897322, "answer_count": 1, "score": 0, "last_activity_date": 1650141240, "creation_date": 1649259694, "last_edit_date": 1650136906, "question_id": 71769754, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/71769754/how-to-fix-the-error-where-the-target-batch-size-does-not-match-when-i-use-cross", "title": "How to fix the error where the target batch size does not match when I use CrossEntropyLoss function?"}, {"tags": ["pytorch", "classification", "cross-entropy"], "owner": {"account_id": 16042323, "reputation": 3, "user_id": 11578461, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/ed667f02c74a4fbe6830bf05cf49a62d?s=256&d=identicon&r=PG&f=1", "display_name": "Ljackson", "link": "https://stackoverflow.com/users/11578461/ljackson"}, "is_answered": true, "view_count": 1180, "accepted_answer_id": 62672126, "answer_count": 1, "score": 0, "last_activity_date": 1593773954, "creation_date": 1593530909, "question_id": 62660950, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/62660950/pytorch-multi-class-classification-using-crossentropyloss-not-converging", "title": "PyTorch Multi Class Classification using CrossEntropyLoss - not converging"}, {"tags": ["pytorch", "classification", "cross-entropy"], "owner": {"account_id": 12461437, "reputation": 542, "user_id": 9072940, "user_type": "registered", "profile_image": "https://lh6.googleusercontent.com/-N9gQOsgQ43Y/AAAAAAAAAAI/AAAAAAAAEBY/Goyr7ha8qXw/photo.jpg?sz=256", "display_name": "Arkadip Bhattacharya", "link": "https://stackoverflow.com/users/9072940/arkadip-bhattacharya"}, "is_answered": true, "view_count": 645, "answer_count": 2, "score": 0, "last_activity_date": 1649039000, "creation_date": 1610378195, "last_edit_date": 1610389741, "question_id": 65669511, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/65669511/difference-between-crossentropyloss-and-nnlloss-with-log-softmax-in-pytorch", "title": "Difference between CrossEntropyLoss and NNLLoss with log_softmax in PyTorch?"}, {"tags": ["pytorch"], "owner": {"account_id": 20591957, "reputation": 3, "user_id": 15115358, "user_type": "registered", "profile_image": "https://lh3.googleusercontent.com/-J8aXxx6YxK4/AAAAAAAAAAI/AAAAAAAAAAA/AMZuuck7KmadwHPspYlqi5X2OIz7DCFSMQ/s96-c/photo.jpg?sz=256", "display_name": "Aser33", "link": "https://stackoverflow.com/users/15115358/aser33"}, "is_answered": false, "view_count": 198, "answer_count": 1, "score": 0, "last_activity_date": 1615537695, "creation_date": 1615530227, "question_id": 66595177, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/66595177/understanding-final-layer-of-output-for-classification-problem-bceloss-vs-crosse", "title": "understanding final layer of output for classification problem BCELoss vs CrossEntropyLoss"}, {"tags": ["python", "neural-network", "pytorch", "softmax", "cross-entropy"], "owner": {"account_id": 10370940, "reputation": 1035, "user_id": 7648650, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/5cfd6a1246b817817a48bd940c57f99b?s=256&d=identicon&r=PG&f=1", "display_name": "Quastiat", "link": "https://stackoverflow.com/users/7648650/quastiat"}, "is_answered": true, "view_count": 3314, "accepted_answer_id": 58123713, "answer_count": 2, "score": 2, "last_activity_date": 1569528338, "creation_date": 1569521478, "last_edit_date": 1569526729, "question_id": 58122505, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/58122505/suppress-use-of-softmax-in-crossentropyloss-for-pytorch-neural-net", "title": "Suppress use of Softmax in CrossEntropyLoss for PyTorch Neural Net"}, {"tags": ["python", "pytorch"], "owner": {"account_id": 20400759, "reputation": 11, "user_id": 14966531, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/0ce6289726b6537541e9b4788cc994ec?s=256&d=identicon&r=PG&f=1", "display_name": "pedrolins", "link": "https://stackoverflow.com/users/14966531/pedrolins"}, "is_answered": true, "view_count": 964, "answer_count": 2, "score": 1, "last_activity_date": 1610440530, "creation_date": 1610117437, "last_edit_date": 1610440530, "question_id": 65631215, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/65631215/input-batch-size-doesnt-match-target-batch-size-in-crossentropyloss-function", "title": "Input batch size doesn&#39;t match target batch size in CrossEntropyLoss function"}, {"tags": ["optimization", "neural-network", "pytorch", "loss-function"], "owner": {"account_id": 10273154, "reputation": 175, "user_id": 7580944, "user_type": "registered", "profile_image": "https://lh6.googleusercontent.com/-ki9iDORYqjE/AAAAAAAAAAI/AAAAAAAAACE/F4owhcdGpAw/photo.jpg?sz=256", "display_name": "Chutlhu", "link": "https://stackoverflow.com/users/7580944/chutlhu"}, "is_answered": false, "view_count": 3807, "answer_count": 1, "score": 5, "last_activity_date": 1604676258, "creation_date": 1539265118, "question_id": 52761533, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/52761533/loss-function-for-ordinal-multi-classification-in-pytorch", "title": "Loss function for ordinal multi classification in pytorch"}, {"tags": ["python", "pytorch", "classification", "weighted", "cross-entropy"], "owner": {"account_id": 23022812, "reputation": 31, "user_id": 17143583, "user_type": "registered", "profile_image": "https://lh3.googleusercontent.com/a/AATXAJw9_qoQ-4oUBZJFI0PfsYbRuBBVP7jxHYyu1BNb=k-s256", "display_name": "patrick823", "link": "https://stackoverflow.com/users/17143583/patrick823"}, "is_answered": true, "view_count": 84, "accepted_answer_id": 70351144, "answer_count": 1, "score": 0, "last_activity_date": 1639494516, "creation_date": 1639488954, "question_id": 70349766, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/70349766/1-weighted-loss-in-crossentropyloss-2-combination-of-weightedrandomsampler-a", "title": "1. Weighted Loss in CrossEntropyLoss() 2. Combination of WeightedRandomSampler and subsampler"}, {"tags": ["pytorch"], "owner": {"account_id": 4512390, "reputation": 3954, "user_id": 3668129, "user_type": "registered", "accept_rate": 49, "profile_image": "https://www.gravatar.com/avatar/?s=256&d=identicon&r=PG&f=1", "display_name": "user3668129", "link": "https://stackoverflow.com/users/3668129/user3668129"}, "is_answered": false, "view_count": 119, "answer_count": 1, "score": 0, "last_activity_date": 1637330996, "creation_date": 1637313970, "question_id": 70032561, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/70032561/why-im-getting-error-when-using-bceloss-when-using-crossentropyloss-all-works", "title": "Why I&#39;m getting error when using BCELoss (when using CrossEntropyLoss all works with no errors)?"}, {"tags": ["pytorch"], "owner": {"account_id": 10841131, "reputation": 961, "user_id": 7972317, "user_type": "registered", "accept_rate": 29, "profile_image": "https://www.gravatar.com/avatar/8f1f825eaa5a35923f6e579634e2821d?s=256&d=identicon&r=PG&f=1", "display_name": "Moran Reznik", "link": "https://stackoverflow.com/users/7972317/moran-reznik"}, "is_answered": true, "view_count": 501, "accepted_answer_id": 67927332, "answer_count": 1, "score": 0, "last_activity_date": 1662740444, "creation_date": 1623349011, "question_id": 67926524, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/67926524/pytorchs-crossentropyloss-how-to-deal-with-the-sequence-length-dimension-with", "title": "PyTorch&#39;s CrossEntropyLoss - how to deal with the sequence length dimension with transformers?"}, {"tags": ["pytorch", "loss", "index-error"], "owner": {"account_id": 13806412, "reputation": 1236, "user_id": 9965155, "user_type": "registered", "profile_image": "https://i.stack.imgur.com/VdWBQ.jpg?s=256&g=1", "display_name": "Anjani Dhrangadhariya", "link": "https://stackoverflow.com/users/9965155/anjani-dhrangadhariya"}, "is_answered": true, "view_count": 2112, "accepted_answer_id": 61502109, "answer_count": 1, "score": 1, "last_activity_date": 1588164603, "creation_date": 1588161893, "question_id": 61501417, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/61501417/input-dimension-for-crossentropy-loss-in-pytorch", "title": "Input dimension for CrossEntropy Loss in PyTorch"}, {"tags": ["pytorch", "artificial-intelligence"], "owner": {"account_id": 12972833, "reputation": 199, "user_id": 9377945, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/5c212c7c49fed1795a8423e6db617947?s=256&d=identicon&r=PG&f=1", "display_name": "ImCoder", "link": "https://stackoverflow.com/users/9377945/imcoder"}, "is_answered": true, "view_count": 32, "accepted_answer_id": 61309830, "answer_count": 1, "score": -1, "last_activity_date": 1587322242, "creation_date": 1587320485, "question_id": 61309456, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/61309456/artificial-neural-networks-which-loss-function-is-used-for-the-optimization-of", "title": "Artificial Neural Networks | Which loss function is used for the optimization of the model?"}, {"tags": ["pytorch", "loss-function", "softmax", "cross-entropy"], "owner": {"account_id": 6273731, "reputation": 189, "user_id": 4877104, "user_type": "registered", "profile_image": "https://graph.facebook.com/10206496849930763/picture?type=large", "display_name": "maccN", "link": "https://stackoverflow.com/users/4877104/maccn"}, "is_answered": true, "view_count": 3046, "accepted_answer_id": 60938973, "answer_count": 1, "score": 1, "last_activity_date": 1585606038, "creation_date": 1585598523, "question_id": 60938630, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/60938630/multi-class-classifcation-with-pytorch", "title": "Multi class classifcation with Pytorch"}, {"tags": ["python", "tensorflow", "pytorch", "loss", "cross-entropy"], "owner": {"account_id": 15085776, "reputation": 51, "user_id": 10887053, "user_type": "registered", "profile_image": "https://lh5.googleusercontent.com/-nLDHwhZhSVA/AAAAAAAAAAI/AAAAAAAAAR4/pBrc9I0qVlg/photo.jpg?sz=256", "display_name": "ankur singhania", "link": "https://stackoverflow.com/users/10887053/ankur-singhania"}, "is_answered": true, "view_count": 245, "accepted_answer_id": 72622279, "answer_count": 1, "score": 1, "last_activity_date": 1655235323, "creation_date": 1655233712, "question_id": 72622202, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/72622202/why-is-the-tensorflow-and-pytorch-crossentropy-loss-returns-different-values-for", "title": "Why is the Tensorflow and Pytorch CrossEntropy loss returns different values for same example"}, {"tags": ["pytorch", "lstm", "word2vec"], "owner": {"account_id": 9609285, "reputation": 550, "user_id": 7134235, "user_type": "registered", "accept_rate": 82, "profile_image": "https://www.gravatar.com/avatar/ee9d8232e77d549dcceae63e2e57b735?s=256&d=identicon&r=PG&f=1", "display_name": "Boris", "link": "https://stackoverflow.com/users/7134235/boris"}, "is_answered": true, "view_count": 1498, "accepted_answer_id": 58104247, "answer_count": 1, "score": 0, "last_activity_date": 1569526112, "creation_date": 1569422738, "question_id": 58101091, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/58101091/choosing-loss-function-for-lstm-trained-on-word2vec-vectors-when-target-is-a-vec", "title": "Choosing loss function for lstm trained on word2vec vectors when target is a vector of same dimensions"}, {"tags": ["python", "pytorch", "cross-entropy"], "owner": {"account_id": 4097449, "reputation": 487, "user_id": 3363813, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/?s=256&d=identicon&r=PG&f=1", "display_name": "user3363813", "link": "https://stackoverflow.com/users/3363813/user3363813"}, "is_answered": true, "view_count": 9460, "answer_count": 2, "score": 7, "last_activity_date": 1625665680, "creation_date": 1587749371, "question_id": 61414065, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/61414065/pytorch-weight-in-cross-entropy-loss", "title": "Pytorch: Weight in cross entropy loss"}, {"tags": ["nlp", "pytorch", "computer-vision", "loss-function", "transformer-model"], "owner": {"account_id": 6263589, "reputation": 142, "user_id": 4870011, "user_type": "registered", "profile_image": "https://lh3.googleusercontent.com/-1awbTdaKdLk/AAAAAAAAAAI/AAAAAAAAI-I/Zi1_1BcUnMw/photo.jpg?sz=256", "display_name": "Vaibhav Sah", "link": "https://stackoverflow.com/users/4870011/vaibhav-sah"}, "is_answered": false, "view_count": 170, "answer_count": 1, "score": 0, "last_activity_date": 1659967077, "creation_date": 1628490947, "question_id": 68707747, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/68707747/vivit-pytorch-runtimeerror-multi-target-not-supported-at-pytorch-aten-src-thc", "title": "ViVIT PyTorch: RuntimeError: multi-target not supported at /pytorch/aten/src/THCUNN/generic/ClassNLLCriterion.cu:15"}, {"tags": ["python", "machine-learning", "torch", "pytorch", "language-model"], "owner": {"account_id": 11601355, "reputation": 127, "user_id": 8498359, "user_type": "registered", "accept_rate": 25, "profile_image": "https://i.stack.imgur.com/YZurj.jpg?s=256&g=1", "display_name": "Sun Chuanneng", "link": "https://stackoverflow.com/users/8498359/sun-chuanneng"}, "is_answered": false, "view_count": 693, "answer_count": 1, "score": 0, "last_activity_date": 1506697721, "creation_date": 1506617384, "last_edit_date": 1506660868, "question_id": 46474426, "content_license": "CC BY-SA 3.0", "link": "https://stackoverflow.com/questions/46474426/assertion-thindextensor-sizetarget-0-batch-size-failed", "title": "Assertion `THIndexTensor_(size)(target, 0) == batch_size&#39; failed"}, {"tags": ["python", "keras", "deep-learning", "pytorch"], "owner": {"account_id": 444297, "reputation": 9927, "user_id": 836026, "user_type": "registered", "accept_rate": 99, "profile_image": "https://www.gravatar.com/avatar/223fb6c30bd49f4df1ac419cd0747a42?s=256&d=identicon&r=PG", "display_name": "user836026", "link": "https://stackoverflow.com/users/836026/user836026"}, "is_answered": true, "view_count": 315, "answer_count": 1, "score": 0, "last_activity_date": 1647242703, "creation_date": 1647225544, "last_edit_date": 1647239172, "question_id": 71462468, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/71462468/altering-pytorch-resnet-head-from-sigmoid-to-softmax", "title": "Altering pytorch resnet head from sigmoid to Softmax"}, {"tags": ["python", "pycharm", "pytorch"], "owner": {"account_id": 17973805, "reputation": 33, "user_id": 13061903, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/f130457e27e574777ec0f85ae4055cfb?s=256&d=identicon&r=PG&f=1", "display_name": "feiyuanyu", "link": "https://stackoverflow.com/users/13061903/feiyuanyu"}, "is_answered": false, "view_count": 165, "answer_count": 1, "score": 0, "last_activity_date": 1585035896, "creation_date": 1584876279, "last_edit_date": 1585035896, "question_id": 60798712, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/60798712/when-i-train-my-network-it-got-an-errror-expected-target-size-4-224-got-to", "title": "when I train my network ,It got an errror, Expected target size (4, 224), got torch.Size([4, 224, 224])"}, {"tags": ["nlp", "pytorch", "text-classification"], "owner": {"account_id": 5140746, "reputation": 11, "user_id": 4118209, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/531a11e85d1c216a3214b7a1500d5edd?s=256&d=identicon&r=PG&f=1", "display_name": "Vinitra", "link": "https://stackoverflow.com/users/4118209/vinitra"}, "is_answered": false, "view_count": 17, "answer_count": 1, "score": 0, "last_activity_date": 1641824944, "creation_date": 1641807577, "question_id": 70650355, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/70650355/loss-not-changing-after-epochs", "title": "Loss not changing after epochs"}, {"tags": ["python", "deep-learning"], "owner": {"account_id": 18231020, "reputation": 45, "user_id": 13269950, "user_type": "registered", "profile_image": "https://i.stack.imgur.com/p6qeN.png?s=256&g=1", "display_name": "likyoo", "link": "https://stackoverflow.com/users/13269950/likyoo"}, "is_answered": true, "view_count": 448, "accepted_answer_id": 61135568, "answer_count": 1, "score": 0, "last_activity_date": 1586502680, "creation_date": 1586440594, "last_edit_date": 1586485452, "question_id": 61122824, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/61122824/when-using-the-cross-entropy-function-for-binary-classification-a-big-gap-betwe", "title": "When using the cross-entropy function for binary classification, a big gap between the model output scalar and a two-dimensional vector"}, {"tags": ["python", "deep-learning", "pytorch", "mlp"], "owner": {"account_id": 21083865, "reputation": 31, "user_id": 15497482, "user_type": "registered", "profile_image": "https://lh4.googleusercontent.com/-4UeTYIgBo3U/AAAAAAAAAAI/AAAAAAAAAAA/AMZuucliC2OhtXiV0Bwl_TcZSQmdEUpYxQ/s96-c/photo.jpg?sz=256", "display_name": "Pemba Lama", "link": "https://stackoverflow.com/users/15497482/pemba-lama"}, "is_answered": false, "view_count": 368, "answer_count": 1, "score": 0, "last_activity_date": 1616963710, "creation_date": 1616901881, "question_id": 66838131, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/66838131/dimension-out-of-range-expected-to-be-in-range-of-4-3-but-got-64", "title": "Dimension out of range (expected to be in range of [-4, 3], but got 64)"}, {"tags": ["python", "pytorch"], "owner": {"account_id": 10437775, "reputation": 143, "user_id": 7695515, "user_type": "registered", "profile_image": "https://i.stack.imgur.com/C0oyh.jpg?s=256&g=1", "display_name": "Anonymous", "link": "https://stackoverflow.com/users/7695515/anonymous"}, "is_answered": false, "view_count": 783, "answer_count": 1, "score": 2, "last_activity_date": 1558617789, "creation_date": 1558595759, "last_edit_date": 1558617789, "question_id": 56269922, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/56269922/nn-crossentropyloss-function-results-in-torch-floattensor-has-no-requires-gra", "title": "nn.CrossEntropyLoss() function results in torch.FloatTensor has no &#39;requires_gradient&#39; attribute error"}, {"tags": ["python", "pytorch", "loss-function"], "owner": {"account_id": 10167937, "reputation": 219, "user_id": 7509178, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/ffd3499c714131cdbe0977bed74ccb82?s=256&d=identicon&r=PG&f=1", "display_name": "justin_sakong", "link": "https://stackoverflow.com/users/7509178/justin-sakong"}, "is_answered": true, "view_count": 784, "accepted_answer_id": 53571764, "answer_count": 1, "score": 1, "last_activity_date": 1543674562, "creation_date": 1543673397, "last_edit_date": 1543674258, "question_id": 53571621, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/53571621/pytorch-crossentropy-error-in-simple-example-of-nn", "title": "pytorch Crossentropy error in simple example of NN"}, {"tags": ["python", "neural-network", "pytorch"], "owner": {"account_id": 25176382, "reputation": 3, "user_id": 19018503, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/4caf0195ca7de7f44aaeb11ef326761f?s=256&d=identicon&r=PG", "display_name": "Litlon", "link": "https://stackoverflow.com/users/19018503/litlon"}, "is_answered": true, "view_count": 175, "accepted_answer_id": 72096597, "answer_count": 1, "score": -1, "last_activity_date": 1651566060, "creation_date": 1651520339, "question_id": 72091572, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/72091572/how-to-compute-cross-entropy-loss-for-sequences", "title": "How to compute Cross Entropy Loss for sequences"}, {"tags": ["python-3.x", "deep-learning", "pytorch"], "owner": {"account_id": 6815600, "reputation": 778, "user_id": 5244410, "user_type": "registered", "accept_rate": 37, "profile_image": "https://www.gravatar.com/avatar/c2e125721f74672d38c5dcd66aaf5bb4?s=256&d=identicon&r=PG&f=1", "display_name": "wprins", "link": "https://stackoverflow.com/users/5244410/wprins"}, "is_answered": false, "view_count": 470, "answer_count": 1, "score": 0, "last_activity_date": 1507976222, "creation_date": 1507935865, "question_id": 46739115, "content_license": "CC BY-SA 3.0", "link": "https://stackoverflow.com/questions/46739115/failing-to-train-skipgram-word-embedding-in-pytorch", "title": "Failing to train SkipGram word embedding in Pytorch"}, {"tags": ["python", "tensorflow", "machine-learning", "keras"], "owner": {"account_id": 15929237, "reputation": 11, "user_id": 11494080, "user_type": "registered", "profile_image": "https://lh4.googleusercontent.com/-SoydVzipmZc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rf8m5rnRrB4xYLkb8kfNw6Rlm7VRw/mo/photo.jpg?sz=256", "display_name": "Moe1234", "link": "https://stackoverflow.com/users/11494080/moe1234"}, "is_answered": true, "view_count": 73, "accepted_answer_id": 67319019, "answer_count": 1, "score": 1, "last_activity_date": 1619705176, "creation_date": 1619476975, "last_edit_date": 1619479492, "question_id": 67274671, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/67274671/tensorflow-keras-cross-entropy-loss-with-linear-activation-on-output", "title": "Tensorflow Keras cross entropy loss with linear activation on output"}, {"tags": ["python", "html", "nlp", "pytorch", "recurrent-neural-network"], "owner": {"account_id": 15576748, "reputation": 23, "user_id": 11237667, "user_type": "registered", "profile_image": "https://lh5.googleusercontent.com/-INvTYTZGbSY/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rdKZgymRaYBmMwbN_7ZvUPYpWFlnQ/mo/photo.jpg?sz=256", "display_name": "OverclockRo", "link": "https://stackoverflow.com/users/11237667/overclockro"}, "is_answered": true, "view_count": 76, "accepted_answer_id": 55282797, "answer_count": 1, "score": 0, "last_activity_date": 1553181668, "creation_date": 1553174794, "question_id": 55281499, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/55281499/pytorch-rnn-html-generation", "title": "Pytorch RNN HTML Generation"}, {"tags": ["python", "machine-learning", "pytorch", "transfer-learning"], "owner": {"account_id": 2398603, "reputation": 3419, "user_id": 2096883, "user_type": "registered", "accept_rate": 78, "profile_image": "https://i.stack.imgur.com/fesnr.png?s=256&g=1", "display_name": "Jared Nielsen", "link": "https://stackoverflow.com/users/2096883/jared-nielsen"}, "is_answered": true, "view_count": 34589, "accepted_answer_id": 66773267, "answer_count": 7, "score": 34, "last_activity_date": 1659625099, "creation_date": 1555290870, "last_edit_date": 1617398999, "question_id": 55681502, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/55681502/label-smoothing-in-pytorch", "title": "Label Smoothing in PyTorch"}, {"tags": ["python", "deep-learning", "pytorch", "computer-vision"], "owner": {"account_id": 15475157, "reputation": 31, "user_id": 14375399, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/a7235d7d39d4c3c4069048bb6d4e2ead?s=256&d=identicon&r=PG&f=1", "display_name": "Akash", "link": "https://stackoverflow.com/users/14375399/akash"}, "is_answered": true, "view_count": 52, "answer_count": 1, "score": 0, "last_activity_date": 1657501528, "creation_date": 1657469821, "last_edit_date": 1657470852, "question_id": 72930012, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/72930012/model-training-converges-to-a-fixed-value-of-loss-with-low-accuracy", "title": "Model training converges to a fixed value of loss with low accuracy"}, {"tags": ["python", "pytorch", "loss", "cross-entropy"], "owner": {"account_id": 13811357, "reputation": 163, "user_id": 9969041, "user_type": "registered", "profile_image": "https://i.stack.imgur.com/ZDqWs.jpg?s=256&g=1", "display_name": "imharjyotbagga", "link": "https://stackoverflow.com/users/9969041/imharjyotbagga"}, "is_answered": true, "view_count": 284, "answer_count": 1, "score": 2, "last_activity_date": 1625493431, "creation_date": 1625487797, "last_edit_date": 1625489755, "question_id": 68256087, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/68256087/pytorch-runtimeerror-expected-object-of-scalar-type-long-but-got-scalar-type", "title": "Pytorch - RuntimeError: Expected object of scalar type Long but got scalar type Float for argument #2 &#39;target&#39; in call to _thnn_nll_loss_forward"}, {"tags": ["python", "machine-learning", "deep-learning", "pytorch", "cross-entropy"], "owner": {"account_id": 11981281, "reputation": 107, "user_id": 8767101, "user_type": "registered", "profile_image": "https://i.stack.imgur.com/qNFex.png?s=256&g=1", "display_name": "miditower", "link": "https://stackoverflow.com/users/8767101/miditower"}, "is_answered": true, "view_count": 733, "answer_count": 1, "score": 0, "last_activity_date": 1555098509, "creation_date": 1555073086, "last_edit_date": 1555074021, "question_id": 55651920, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/55651920/how-to-use-torch-nn-crossentropyloss-as-autoencoders-reconstruction-loss", "title": "How to use torch.nn.CrossEntropyLoss as autoencoder&#39;s reconstruction loss?"}, {"tags": ["python", "pytorch", "one-hot-encoding", "cross-entropy"], "owner": {"account_id": 19437334, "reputation": 25, "user_id": 14571158, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/7b1604063246a569eddb056697081681?s=256&d=identicon&r=PG&f=1", "display_name": "joni", "link": "https://stackoverflow.com/users/14571158/joni"}, "is_answered": true, "view_count": 3008, "answer_count": 1, "score": 1, "last_activity_date": 1663614248, "creation_date": 1606651145, "last_edit_date": 1663614174, "question_id": 65059829, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/65059829/pytorch-categorical-cross-entropy-loss-using-one-hot-encoding-and-softmax", "title": "Pytorch - (Categorical) Cross Entropy Loss using one hot encoding and softmax"}, {"tags": ["python", "numpy", "pytorch"], "owner": {"account_id": 19145247, "reputation": 25, "user_id": 13984381, "user_type": "registered", "profile_image": "https://lh6.googleusercontent.com/-Bu2ySOgmQbs/AAAAAAAAAAI/AAAAAAAAAAA/AMZuucnw4cnYo9f9-PKusP_vKFyZ10YGVw/photo.jpg?sz=256", "display_name": "Luke Hoffman", "link": "https://stackoverflow.com/users/13984381/luke-hoffman"}, "is_answered": true, "view_count": 639, "accepted_answer_id": 64957574, "answer_count": 1, "score": -1, "last_activity_date": 1606067685, "creation_date": 1606064629, "question_id": 64957007, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/64957007/python-error-runtimeerror-expected-scalar-type-long-but-found-double", "title": "Python Error RuntimeError: expected scalar type Long but found Double"}, {"tags": ["python", "deep-learning", "neural-network", "pytorch", "semantic-segmentation"], "owner": {"account_id": 20730559, "reputation": 37, "user_id": 15222682, "user_type": "registered", "profile_image": "https://lh4.googleusercontent.com/-hK7EKn1lesw/AAAAAAAAAAI/AAAAAAAAAAA/AMZuucmZ2roZyys9sL0ETcd3365E7cQc6A/s96-c/photo.jpg?sz=256", "display_name": "Alisove", "link": "https://stackoverflow.com/users/15222682/alisove"}, "is_answered": false, "view_count": 112, "answer_count": 1, "score": 0, "last_activity_date": 1653378474, "creation_date": 1653331217, "question_id": 72353412, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/72353412/segmentation-loss-function", "title": "Segmentation loss function"}, {"tags": ["pytorch", "vgg-net"], "owner": {"account_id": 12195912, "reputation": 31, "user_id": 8901565, "user_type": "registered", "profile_image": "https://lh6.googleusercontent.com/-f1kPCp9LizM/AAAAAAAAAAI/AAAAAAAAAs4/szMTsrMB_mw/photo.jpg?sz=256", "display_name": "Ricard Borr&#224;s", "link": "https://stackoverflow.com/users/8901565/ricard-borr%c3%a0s"}, "is_answered": false, "view_count": 1228, "answer_count": 1, "score": 3, "last_activity_date": 1570907214, "creation_date": 1570821989, "last_edit_date": 1570822345, "question_id": 58347382, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/58347382/1x1-convolution-as-classification-layer-in-pytorch", "title": "1x1 convolution as classification layer in Pytorch"}, {"tags": ["python", "machine-learning", "deep-learning", "neural-network", "pytorch"], "owner": {"account_id": 21367901, "reputation": 37, "user_id": 15735826, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/e07b48e89c03f02b943e0f81fdec31f4?s=256&d=identicon&r=PG&f=1", "display_name": "marv_97", "link": "https://stackoverflow.com/users/15735826/marv-97"}, "is_answered": true, "view_count": 230, "accepted_answer_id": 71048897, "answer_count": 1, "score": 0, "last_activity_date": 1644406659, "creation_date": 1644403554, "last_edit_date": 1644403995, "question_id": 71048172, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/71048172/difference-between-the-calculation-of-the-training-loss-and-validation-loss-usin", "title": "Difference between the calculation of the training loss and validation loss using pytorch"}, {"tags": ["coreml", "cross-entropy", "coremltools", "mlmodel"], "owner": {"account_id": 15379113, "reputation": 55, "user_id": 11095013, "user_type": "registered", "profile_image": "https://lh6.googleusercontent.com/-j7NAmEr0n60/AAAAAAAAAAI/AAAAAAAAABM/92rmpHQ7sqo/photo.jpg?sz=256", "display_name": "YiZhaoYanBo", "link": "https://stackoverflow.com/users/11095013/yizhaoyanbo"}, "is_answered": false, "view_count": 65, "answer_count": 1, "score": 0, "last_activity_date": 1584044282, "creation_date": 1584006754, "question_id": 60651480, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/60651480/i-am-confused-to-make-mlmodel-updatable-using-coreml3-tools", "title": "I am confused to make mlmodel updatable using coreml3 tools"}, {"tags": ["python", "deep-learning", "pytorch"], "owner": {"account_id": 6815600, "reputation": 778, "user_id": 5244410, "user_type": "registered", "accept_rate": 37, "profile_image": "https://www.gravatar.com/avatar/c2e125721f74672d38c5dcd66aaf5bb4?s=256&d=identicon&r=PG&f=1", "display_name": "wprins", "link": "https://stackoverflow.com/users/5244410/wprins"}, "is_answered": true, "view_count": 4156, "answer_count": 1, "score": 0, "last_activity_date": 1512262336, "creation_date": 1511997019, "last_edit_date": 1512262336, "question_id": 47563398, "content_license": "CC BY-SA 3.0", "link": "https://stackoverflow.com/questions/47563398/loss-dimensionality-issue-in-pytorch-sequence-to-label-learning", "title": "Loss dimensionality issue in PyTorch (sequence to label learning)"}, {"tags": ["python", "pytorch", "naming-conventions", "convolution", "cross-correlation"], "owner": {"account_id": 3559086, "reputation": 15523, "user_id": 9067615, "user_type": "registered", "profile_image": "https://i.stack.imgur.com/U7ZMS.jpg?s=256&g=1", "display_name": "iacob", "link": "https://stackoverflow.com/users/9067615/iacob"}, "is_answered": true, "view_count": 799, "accepted_answer_id": 66641773, "answer_count": 1, "score": 1, "last_activity_date": 1615927874, "creation_date": 1615821657, "last_edit_date": 1615823979, "question_id": 66640802, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/66640802/why-are-pytorch-convolutions-implemented-as-cross-correlations", "title": "Why are PyTorch &quot;convolutions&quot; implemented as cross-correlations?"}, {"tags": ["python", "pytorch"], "owner": {"account_id": 19048055, "reputation": 19, "user_id": 16573587, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/8d2c7e6a68d1ad3e8a9537fe75f2f4b2?s=256&d=identicon&r=PG&f=1", "display_name": "user16573587", "link": "https://stackoverflow.com/users/16573587/user16573587"}, "is_answered": true, "view_count": 573, "accepted_answer_id": 69560026, "answer_count": 1, "score": 0, "last_activity_date": 1634146961, "creation_date": 1634146500, "question_id": 69559925, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/69559925/what-is-the-predicted-output-label-from-a-pytorch-model", "title": "What is the predicted output label from a PyTorch model?"}, {"tags": ["python", "neural-network", "pytorch", "image-segmentation", "cross-entropy"], "owner": {"account_id": 10933694, "reputation": 201, "user_id": 8036071, "user_type": "registered", "accept_rate": 0, "profile_image": "https://lh6.googleusercontent.com/-wZCId_aQcM4/AAAAAAAAAAI/AAAAAAAAABA/wbYbkDUxirk/photo.jpg?sz=256", "display_name": "Rajesh", "link": "https://stackoverflow.com/users/8036071/rajesh"}, "is_answered": true, "view_count": 1131, "answer_count": 1, "score": 0, "last_activity_date": 1619722350, "creation_date": 1619719503, "question_id": 67322848, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/67322848/unet-with-crossentropy-loss-function", "title": "UNET with CrossEntropy Loss Function"}, {"tags": ["deep-learning", "neural-network", "nlp", "pytorch", "nlp-question-answering"], "owner": {"account_id": 18881159, "reputation": 1, "user_id": 13773521, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/7fd2bdc94b53c2f24892bca1dd07dc0c?s=256&d=identicon&r=PG&f=1", "display_name": "Kushagra Bhatia", "link": "https://stackoverflow.com/users/13773521/kushagra-bhatia"}, "is_answered": false, "view_count": 238, "answer_count": 1, "score": 0, "last_activity_date": 1592734661, "creation_date": 1592672452, "last_edit_date": 1592675510, "question_id": 62488968, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/62488968/combine-two-tensors-of-same-dimension-to-get-a-final-output-tensor-using-trainab", "title": "Combine two tensors of same dimension to get a final output tensor using trainable weights"}, {"tags": ["pytorch", "sequence", "loss-function", "cross-entropy"], "owner": {"account_id": 7261167, "reputation": 7659, "user_id": 5537840, "user_type": "registered", "accept_rate": 70, "profile_image": "https://i.stack.imgur.com/aKRoy.jpg?s=256&g=1", "display_name": "Kenenbek Arzymatov", "link": "https://stackoverflow.com/users/5537840/kenenbek-arzymatov"}, "is_answered": true, "view_count": 604, "answer_count": 1, "score": 3, "last_activity_date": 1632860415, "creation_date": 1632858306, "last_edit_date": 1632860415, "question_id": 69367671, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/69367671/correct-use-of-cross-entropy-as-a-loss-function-for-sequence-of-elements", "title": "Correct use of Cross-entropy as a loss function for sequence of elements"}, {"tags": ["pytorch", "classification", "multiclass-classification", "softmax", "cross-entropy"], "owner": {"account_id": 23022812, "reputation": 31, "user_id": 17143583, "user_type": "registered", "profile_image": "https://lh3.googleusercontent.com/a/AATXAJw9_qoQ-4oUBZJFI0PfsYbRuBBVP7jxHYyu1BNb=k-s256", "display_name": "patrick823", "link": "https://stackoverflow.com/users/17143583/patrick823"}, "is_answered": true, "view_count": 691, "answer_count": 2, "score": 1, "last_activity_date": 1640635439, "creation_date": 1639133982, "question_id": 70303466, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/70303466/do-i-need-to-apply-the-softmax-function-anywhere-in-my-multi-class-classificatio", "title": "Do I need to apply the Softmax Function ANYWHERE in my multi-class classification Model?"}, {"tags": ["machine-learning", "deep-learning", "pytorch", "conv-neural-network", "resnet"], "owner": {"account_id": 18601951, "reputation": 1, "user_id": 13556577, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/94bd462dc7c4553c8257dfcb9654233b?s=256&d=identicon&r=PG&f=1", "display_name": "KiriNi", "link": "https://stackoverflow.com/users/13556577/kirini"}, "is_answered": false, "view_count": 682, "answer_count": 1, "score": 0, "last_activity_date": 1589707982, "creation_date": 1589654846, "last_edit_date": 1589657042, "question_id": 61841938, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/61841938/how-can-i-improve-my-pytorch-implementation-of-resnet-for-cifar-10-classificatio", "title": "How can I improve my PyTorch implementation of ResNet for CIFAR-10 classification?"}, {"tags": ["deep-learning", "neural-network", "pytorch", "multitasking"], "owner": {"account_id": 6233765, "reputation": 2241, "user_id": 5112804, "user_type": "registered", "accept_rate": 62, "profile_image": "https://i.stack.imgur.com/YLPIr.png?s=256&g=1", "display_name": "inverted_index", "link": "https://stackoverflow.com/users/5112804/inverted-index"}, "is_answered": true, "view_count": 449, "accepted_answer_id": 61583814, "answer_count": 1, "score": 0, "last_activity_date": 1588555441, "creation_date": 1588541148, "last_edit_date": 1588542997, "question_id": 61581839, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/61581839/the-loss-effects-in-multitask-learning-framework", "title": "The loss effects in multitask learning framework"}, {"tags": ["pytorch"], "owner": {"account_id": 7167400, "reputation": 744, "user_id": 8111837, "user_type": "registered", "accept_rate": 62, "profile_image": "https://i.stack.imgur.com/mwoa1.jpg?s=256&g=1", "display_name": "Alexander Pugantsov", "link": "https://stackoverflow.com/users/8111837/alexander-pugantsov"}, "is_answered": true, "view_count": 347, "accepted_answer_id": 66260850, "answer_count": 1, "score": 0, "last_activity_date": 1613654575, "creation_date": 1613652825, "question_id": 66260394, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/66260394/pytorch-reshape-1d-to-2d-target", "title": "Pytorch: Reshape 1d to 2d target"}, {"tags": ["python", "neural-network", "pytorch"], "owner": {"account_id": 12394026, "reputation": 1409, "user_id": 9464073, "user_type": "registered", "profile_image": "https://i.stack.imgur.com/YOTvg.png?s=256&g=1", "display_name": "Mr. President", "link": "https://stackoverflow.com/users/9464073/mr-president"}, "is_answered": true, "view_count": 60, "accepted_answer_id": 57842425, "answer_count": 1, "score": 1, "last_activity_date": 1567949623, "creation_date": 1567949350, "question_id": 57842400, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/57842400/understanding-the-use-of-classes-when-training-a-neural-network-with-pytorch", "title": "Understanding the use of classes when training a neural network with Pytorch"}, {"tags": ["python", "neural-network", "pytorch"], "owner": {"account_id": 8651023, "reputation": 73, "user_id": 6476721, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/7d9411aae0b9b2a3163c6cb4263edc3f?s=256&d=identicon&r=PG&f=1", "display_name": "manestay", "link": "https://stackoverflow.com/users/6476721/manestay"}, "is_answered": true, "view_count": 80, "answer_count": 1, "score": 1, "last_activity_date": 1532106276, "creation_date": 1532049428, "last_edit_date": 1532106276, "question_id": 51433774, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/51433774/basic-dnn-with-highly-imbalanced-dataset-network-predicts-same-labels", "title": "Basic DNN with highly imbalanced dataset -- network predicts same labels"}, {"tags": ["python", "machine-learning", "deep-learning", "pytorch", "pytorch-lightning"], "owner": {"account_id": 543843, "reputation": 18837, "user_id": 913098, "user_type": "registered", "accept_rate": 50, "profile_image": "https://www.gravatar.com/avatar/e3e07b09128f889209b738276d42d7ef?s=256&d=identicon&r=PG", "display_name": "Gulzar", "link": "https://stackoverflow.com/users/913098/gulzar"}, "is_answered": true, "view_count": 704, "accepted_answer_id": 67017727, "answer_count": 1, "score": 0, "last_activity_date": 1617957272, "creation_date": 1617877692, "last_edit_date": 1617878041, "question_id": 67002099, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/67002099/how-to-use-pytorch-lightning-accuracy-with-ignore-class", "title": "How to use pytorch lightning Accuracy with ignore class?"}, {"tags": ["huggingface-transformers"], "owner": {"account_id": 3476569, "reputation": 63, "user_id": 2909599, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/a530518320502fcf25ddab94ec54abf3?s=256&d=identicon&r=PG&f=1", "display_name": "jinki", "link": "https://stackoverflow.com/users/2909599/jinki"}, "is_answered": false, "view_count": 233, "answer_count": 1, "score": 2, "last_activity_date": 1632317731, "creation_date": 1580276177, "question_id": 59961023, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/59961023/is-sopsentence-order-prediction-implemented", "title": "is SOP(sentence order prediction) implemented?"}, {"tags": ["python", "tensorflow", "neural-network", "loss", "tf-slim"], "owner": {"account_id": 10932857, "reputation": 489, "user_id": 8035425, "user_type": "registered", "accept_rate": 33, "profile_image": "https://www.gravatar.com/avatar/45b1610bc8e203db35d233a5280b9fd8?s=256&d=identicon&r=PG&f=1", "display_name": "dooder", "link": "https://stackoverflow.com/users/8035425/dooder"}, "is_answered": true, "view_count": 6597, "accepted_answer_id": 44065956, "answer_count": 3, "score": 2, "last_activity_date": 1495341308, "creation_date": 1495181799, "question_id": 44064859, "content_license": "CC BY-SA 3.0", "link": "https://stackoverflow.com/questions/44064859/tensorflow-neural-network-loss-value-nan", "title": "Tensorflow neural network loss value NaN"}, {"tags": ["deep-learning", "neural-network", "pytorch", "regression", "classification"], "owner": {"account_id": 24760407, "reputation": 11, "user_id": 18644883, "user_type": "registered", "profile_image": "https://graph.facebook.com/10159220745954998/picture?type=large", "display_name": "Felipe Bayona", "link": "https://stackoverflow.com/users/18644883/felipe-bayona"}, "is_answered": true, "view_count": 43, "answer_count": 1, "score": 1, "last_activity_date": 1648775582, "creation_date": 1648738930, "last_edit_date": 1648775582, "question_id": 71694867, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/71694867/how-can-i-implement-a-simple-dot-counting-problem-using-a-regression-instead-of", "title": "How can I implement a simple dot counting problem using a regression instead of a classification approach in Pytorch?"}, {"tags": ["pytorch", "google-colaboratory", "resnet"], "owner": {"account_id": 17540480, "reputation": 1, "user_id": 12722732, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/d07c29d85ffedca7bd4d079f1e2f8b95?s=256&d=identicon&r=PG&f=1", "display_name": "pang54", "link": "https://stackoverflow.com/users/12722732/pang54"}, "is_answered": true, "view_count": 283, "accepted_answer_id": 66434832, "answer_count": 1, "score": 0, "last_activity_date": 1614669198, "creation_date": 1614445939, "last_edit_date": 1614446282, "question_id": 66401357, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/66401357/pytorch-resnet-model-error-if-fc-layer-is-changed-in-colab", "title": "Pytorch Resnet model error if FC layer is changed in Colab"}, {"tags": ["deep-learning", "pytorch"], "owner": {"account_id": 10332017, "reputation": 19, "user_id": 7621488, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/d37520faa2b7356731fe62f3b78caa5b?s=256&d=identicon&r=PG&f=1", "display_name": "Austin Moon", "link": "https://stackoverflow.com/users/7621488/austin-moon"}, "is_answered": false, "view_count": 87, "answer_count": 1, "score": 1, "last_activity_date": 1592607420, "creation_date": 1592556506, "question_id": 62466618, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/62466618/how-to-train-multiple-losses-in-one-model-with-freezing-some-part-of-the-network", "title": "How to train multiple losses in one model with freezing some part of the network optionally in pytorch?"}, {"tags": ["machine-learning", "huggingface-transformers", "gpt", "perplexity"], "owner": {"account_id": 914564, "reputation": 2345, "user_id": 947659, "user_type": "registered", "accept_rate": 92, "profile_image": "https://www.gravatar.com/avatar/26679042ad95d2683b6de5f886b3d7c3?s=256&d=identicon&r=PG", "display_name": "user947659", "link": "https://stackoverflow.com/users/947659/user947659"}, "is_answered": true, "view_count": 4896, "accepted_answer_id": 60834242, "answer_count": 2, "score": 6, "last_activity_date": 1646228036, "creation_date": 1585058285, "last_edit_date": 1593585558, "question_id": 60832547, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/60832547/where-is-perplexity-calculated-in-the-huggingface-gpt2-language-model-code", "title": "Where is perplexity calculated in the Huggingface gpt2 language model code?"}, {"tags": ["python", "machine-learning", "neural-network", "pytorch", "loss-function"], "owner": {"account_id": 17790524, "reputation": 328, "user_id": 13181562, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/33952ae6b3bc3f19d6597a42d0ba45a0?s=256&d=identicon&r=PG&f=1", "display_name": "Matze", "link": "https://stackoverflow.com/users/13181562/matze"}, "is_answered": true, "view_count": 151, "closed_date": 1640879358, "accepted_answer_id": 70533219, "answer_count": 1, "score": 2, "last_activity_date": 1640875878, "creation_date": 1640874659, "question_id": 70533018, "link": "https://stackoverflow.com/questions/70533018/cross-entropy-yields-different-results-for-vectors-with-identical-distributions", "closed_reason": "Not suitable for this site", "title": "Cross entropy yields different results for vectors with identical distributions"}, {"tags": ["python", "machine-learning", "pytorch"], "owner": {"account_id": 13575957, "reputation": 2753, "user_id": 9793316, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/b327e8b8a1ec719535e7b730ff5cced2?s=256&d=identicon&r=PG&f=1", "display_name": "Raven Cheuk", "link": "https://stackoverflow.com/users/9793316/raven-cheuk"}, "is_answered": true, "view_count": 37522, "accepted_answer_id": 55549952, "answer_count": 4, "score": 26, "last_activity_date": 1654890350, "creation_date": 1554559372, "question_id": 55549843, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/55549843/pytorch-doesnt-support-one-hot-vector", "title": "Pytorch doesn&#39;t support one-hot vector?"}, {"tags": ["python", "pytorch", "google-colaboratory"], "owner": {"account_id": 2644481, "reputation": 557, "user_id": 2287486, "user_type": "registered", "accept_rate": 62, "profile_image": "https://www.gravatar.com/avatar/265c9df814f98b9409bc8cdee1e8c917?s=256&d=identicon&r=PG", "display_name": "khushbu", "link": "https://stackoverflow.com/users/2287486/khushbu"}, "is_answered": false, "view_count": 3341, "answer_count": 1, "score": 2, "last_activity_date": 1566538028, "creation_date": 1566487327, "last_edit_date": 1566493013, "question_id": 57612610, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/57612610/how-to-solve-quotno-module-named-39tools39quot-in-google-colab", "title": "How to solve &amp;quot;No module named &amp;#39;tools&amp;#39;&amp;quot; in google colab?"}, {"tags": ["deep-learning", "pytorch", "loss-function", "cross-entropy"], "owner": {"account_id": 6642002, "reputation": 9703, "user_id": 6210807, "user_type": "registered", "profile_image": "https://lh6.googleusercontent.com/-TFwTfhA1-Eo/AAAAAAAAAAI/AAAAAAAAAJI/FVjF75sxqrc/photo.jpg?sz=256", "display_name": "kHarshit", "link": "https://stackoverflow.com/users/6210807/kharshit"}, "is_answered": true, "view_count": 2649, "accepted_answer_id": 60170298, "answer_count": 1, "score": 4, "last_activity_date": 1581428753, "creation_date": 1581415776, "question_id": 60166427, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/60166427/how-do-i-calculate-cross-entropy-from-probabilities-in-pytorch", "title": "How do I calculate cross-entropy from probabilities in PyTorch?"}, {"tags": ["pytorch"], "owner": {"account_id": 4512390, "reputation": 3954, "user_id": 3668129, "user_type": "registered", "accept_rate": 49, "profile_image": "https://www.gravatar.com/avatar/?s=256&d=identicon&r=PG&f=1", "display_name": "user3668129", "link": "https://stackoverflow.com/users/3668129/user3668129"}, "is_answered": false, "view_count": 775, "answer_count": 1, "score": 0, "last_activity_date": 1637422125, "creation_date": 1637405746, "question_id": 70045245, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/70045245/loss-backward-gives-runtimeerror-cuda-error-device-side-assert-triggered", "title": "loss.backward() gives RuntimeError: CUDA error: device-side assert triggered"}, {"tags": ["python", "machine-learning", "pytorch"], "owner": {"account_id": 1480593, "reputation": 651, "user_id": 1390093, "user_type": "registered", "accept_rate": 89, "profile_image": "https://www.gravatar.com/avatar/a71cc2a6b1240dc5217dbfa471dbf674?s=256&d=identicon&r=PG", "display_name": "mkmitchell", "link": "https://stackoverflow.com/users/1390093/mkmitchell"}, "is_answered": true, "view_count": 32092, "answer_count": 2, "score": 7, "last_activity_date": 1547231016, "creation_date": 1547057668, "last_edit_date": 1547061222, "question_id": 54116080, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/54116080/having-issues-with-neural-network-training-loss-not-decreasing", "title": "Having issues with neural network training. Loss not decreasing"}, {"tags": ["keras", "pytorch", "tensorflow2.0", "bert-language-model", "huggingface-transformers"], "owner": {"account_id": 15991608, "reputation": 19, "user_id": 11540649, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/292fa3b519a318f42fb37a3360098ffe?s=256&d=identicon&r=PG&f=1", "display_name": "Vasu Dev Reddy", "link": "https://stackoverflow.com/users/11540649/vasu-dev-reddy"}, "is_answered": true, "view_count": 2046, "answer_count": 1, "score": 1, "last_activity_date": 1600847096, "creation_date": 1600835035, "question_id": 64020998, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/64020998/getting-an-errorcannot-import-name-39bertpretrainedmodel39-while-importi", "title": "Getting an error(cannot import name &amp;#39;BertPreTrainedModel&amp;#39;) while importing classification model from simpletransformers"}, {"tags": ["python", "conv-neural-network", "pytorch", "multiclass-classification", "cross-entropy"], "owner": {"account_id": 7123660, "reputation": 89, "user_id": 6276294, "user_type": "registered", "accept_rate": 60, "profile_image": "https://lh4.googleusercontent.com/-3Pxl6hHCA3s/AAAAAAAAAAI/AAAAAAAAAE8/9X5NTYxm6Ew/photo.jpg?sz=256", "display_name": "lvl", "link": "https://stackoverflow.com/users/6276294/lvl"}, "is_answered": true, "view_count": 6588, "answer_count": 1, "score": 3, "last_activity_date": 1550124570, "creation_date": 1550095981, "question_id": 54680267, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/54680267/apply-a-pytorch-crossentropy-method-for-multiclass-segmentation", "title": "Apply a PyTorch CrossEntropy method for multiclass segmentation"}, {"tags": ["conv-neural-network", "pixel", "pytorch"], "owner": {"account_id": 15224917, "reputation": 21, "user_id": 10985820, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/c6ce9a62ea220b911ebf6ae8eef434ea?s=256&d=identicon&r=PG&f=1", "display_name": "S.Q", "link": "https://stackoverflow.com/users/10985820/s-q"}, "is_answered": true, "view_count": 1803, "answer_count": 1, "score": 2, "last_activity_date": 1611231932, "creation_date": 1548781872, "last_edit_date": 1550219716, "question_id": 54426268, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/54426268/using-cnn-for-pixel-classification-how", "title": "Using CNN for pixel classification,how?"}, {"tags": ["fb-hydra", "omegaconf"], "owner": {"account_id": 11479836, "reputation": 526, "user_id": 8414280, "user_type": "registered", "profile_image": "https://lh3.googleusercontent.com/-WQ-3lWnPqWU/AAAAAAAAAAI/AAAAAAAALDU/mzmVWP0sBb4/photo.jpg?sz=256", "display_name": "Celso Fran&#231;a", "link": "https://stackoverflow.com/users/8414280/celso-fran%c3%a7a"}, "is_answered": true, "view_count": 75, "answer_count": 1, "score": 0, "last_activity_date": 1646533335, "creation_date": 1646507837, "question_id": 71365286, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/71365286/how-to-properly-generate-a-multi-level-composition", "title": "How to properly generate a multi-level composition?"}, {"tags": ["python", "deep-learning", "pytorch", "pre-trained-model", "natural-language-processing"], "owner": {"account_id": 11634630, "reputation": 805, "user_id": 8521468, "user_type": "registered", "accept_rate": 73, "profile_image": "https://i.stack.imgur.com/W43ng.png?s=256&g=1", "display_name": "jonb", "link": "https://stackoverflow.com/users/8521468/jonb"}, "is_answered": true, "view_count": 2916, "accepted_answer_id": 56303568, "answer_count": 1, "score": 2, "last_activity_date": 1558776738, "creation_date": 1558199282, "last_edit_date": 1558237535, "question_id": 56201147, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/56201147/how-to-access-the-predictions-of-pytorch-classification-model-bert", "title": "How to access the predictions of pytorch classification model? (BERT)"}, {"tags": ["python", "machine-learning", "deep-learning", "pytorch", "loss-function"], "owner": {"account_id": 543843, "reputation": 18837, "user_id": 913098, "user_type": "registered", "accept_rate": 50, "profile_image": "https://www.gravatar.com/avatar/e3e07b09128f889209b738276d42d7ef?s=256&d=identicon&r=PG", "display_name": "Gulzar", "link": "https://stackoverflow.com/users/913098/gulzar"}, "is_answered": false, "view_count": 222, "answer_count": 1, "score": 0, "last_activity_date": 1614918101, "creation_date": 1614877298, "last_edit_date": 1614877992, "question_id": 66479620, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/66479620/how-to-implement-focalloss-in-pytorch", "title": "How to implement FocalLoss in Pytorch?"}, {"tags": ["python", "machine-learning", "deep-learning", "pytorch", "conv-neural-network"], "owner": {"account_id": 16077980, "reputation": 106, "user_id": 12217978, "user_type": "registered", "profile_image": "https://lh3.googleusercontent.com/-foHwQpkOi6I/AAAAAAAAAAI/AAAAAAAABTg/eyyHkeRJnbg/photo.jpg?sz=256", "display_name": "Aditya Kendre", "link": "https://stackoverflow.com/users/12217978/aditya-kendre"}, "is_answered": true, "view_count": 633, "answer_count": 1, "score": 5, "last_activity_date": 1601388002, "creation_date": 1600636293, "last_edit_date": 1601004714, "question_id": 63983710, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/63983710/cnn-model-is-overfitting-to-data-after-reaching-50-accuracy", "title": "CNN model is overfitting to data after reaching 50% accuracy"}, {"tags": ["tensorflow", "machine-learning", "pytorch", "loss-function", "cross-entropy"], "owner": {"account_id": 20475081, "reputation": 151, "user_id": 15024599, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/c9203ae7b4368b4d15935545acca5b62?s=256&d=identicon&r=PG&f=1", "display_name": "Zhongzheng_11", "link": "https://stackoverflow.com/users/15024599/zhongzheng-11"}, "is_answered": true, "view_count": 1948, "accepted_answer_id": 68617367, "answer_count": 3, "score": 2, "last_activity_date": 1640888338, "creation_date": 1627812857, "last_edit_date": 1627815607, "question_id": 68609414, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/68609414/how-to-calculate-correct-cross-entropy-between-2-tensors-in-pytorch-when-target", "title": "How to calculate correct Cross Entropy between 2 tensors in Pytorch when target is not one-hot?"}, {"tags": ["python", "pytorch", "loss-function"], "owner": {"account_id": 17452657, "reputation": 297, "user_id": 12652301, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/acbf419ef076db5a23bd7c60dd5d9c6b?s=256&d=identicon&r=PG&f=1", "display_name": "Xaume", "link": "https://stackoverflow.com/users/12652301/xaume"}, "is_answered": false, "view_count": 310, "answer_count": 1, "score": 1, "last_activity_date": 1647795399, "creation_date": 1647353163, "question_id": 71483639, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/71483639/how-to-use-real-world-weight-cross-entropy-loss-in-pytorch", "title": "How to use Real-World-Weight Cross-Entropy loss in PyTorch"}, {"tags": ["nlp", "pytorch", "transformer-model", "huggingface-transformers", "bert-language-model"], "owner": {"account_id": 14980691, "reputation": 103, "user_id": 10814749, "user_type": "registered", "profile_image": "https://i.stack.imgur.com/2P27P.jpg?s=256&g=1", "display_name": "Kaim hong", "link": "https://stackoverflow.com/users/10814749/kaim-hong"}, "is_answered": true, "view_count": 4524, "accepted_answer_id": 66593509, "answer_count": 1, "score": 9, "last_activity_date": 1615806895, "creation_date": 1595408820, "last_edit_date": 1595412515, "question_id": 63030692, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/63030692/how-do-i-use-bertformaskedlm-or-bertmodel-to-calculate-perplexity-of-a-sentence", "title": "How do I use BertForMaskedLM or BertModel to calculate perplexity of a sentence?"}, {"tags": ["python", "torch", "huggingface-transformers"], "owner": {"account_id": 6602312, "reputation": 678, "user_id": 5100213, "user_type": "registered", "accept_rate": 79, "profile_image": "https://i.stack.imgur.com/nN43X.png?s=256&g=1", "display_name": "timekeeper", "link": "https://stackoverflow.com/users/5100213/timekeeper"}, "is_answered": true, "view_count": 881, "answer_count": 1, "score": 0, "last_activity_date": 1630191144, "creation_date": 1604180779, "question_id": 64626722, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/64626722/importerror-cannot-import-name-softmax-backward-data", "title": "ImportError: cannot import name &#39;_softmax_backward_data&#39;"}, {"tags": ["python", "pytorch", "convergence", "siamese-network"], "owner": {"account_id": 16921477, "reputation": 82, "user_id": 12238187, "user_type": "registered", "profile_image": "https://lh3.googleusercontent.com/a-/AAuE7mDjM7vN-HPp36mEmIdsqgg5GVAcYSHoyOpsbf9M=k-s256", "display_name": "Emile Beukes", "link": "https://stackoverflow.com/users/12238187/emile-beukes"}, "is_answered": true, "view_count": 1072, "accepted_answer_id": 62694063, "answer_count": 2, "score": 4, "last_activity_date": 1593685071, "creation_date": 1589446722, "last_edit_date": 1593333902, "question_id": 61793268, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/61793268/pytorch-siamese-network-not-converging", "title": "Pytorch Siamese Network not converging"}, {"tags": ["python", "numpy", "neural-network", "runtime-error", "nan"], "owner": {"account_id": 18677220, "reputation": 39, "user_id": 13614627, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/75cd8bd33a196a0da5eccf9230a4c4aa?s=256&d=identicon&r=PG&f=1", "display_name": "Sip_", "link": "https://stackoverflow.com/users/13614627/sip"}, "is_answered": false, "view_count": 268, "answer_count": 1, "score": 0, "last_activity_date": 1598519622, "creation_date": 1598467503, "question_id": 63603863, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/63603863/how-to-fix-invalid-value-divide-by-zero-and-other-calculation-errors", "title": "How to fix &quot;invalid value&quot;, &quot;divide by zero&quot; and other calculation errors"}, {"tags": ["pytorch"], "owner": {"account_id": 16591400, "reputation": 37, "user_id": 11989487, "user_type": "registered", "profile_image": "https://lh4.googleusercontent.com/-4UDwVXYOfzc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rc61J_kN-sH6he2Wvkv8i8wyL-10A/photo.jpg?sz=256", "display_name": "fausto wang", "link": "https://stackoverflow.com/users/11989487/fausto-wang"}, "is_answered": true, "view_count": 1376, "accepted_answer_id": 58258490, "answer_count": 1, "score": 1, "last_activity_date": 1570373983, "creation_date": 1570372313, "last_edit_date": 1570372716, "question_id": 58258255, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/58258255/error-in-ret-torch-c-nn-nll-loss2dinput-target-weight-reduction-get-enu", "title": "Error in ret = torch._C._nn.nll_loss2d(input, target, weight, _Reduction.get_enum(reduction), ignore_index)"}, {"tags": ["loss-function", "machine-translation", "returnn"], "owner": {"account_id": 3286986, "reputation": 2154, "user_id": 2766231, "user_type": "registered", "accept_rate": 79, "profile_image": "https://www.gravatar.com/avatar/3c36a95cabe82bbb5557474bcfad6c24?s=256&d=identicon&r=PG&f=1", "display_name": "Frithjof", "link": "https://stackoverflow.com/users/2766231/frithjof"}, "is_answered": true, "view_count": 157, "accepted_answer_id": 63325710, "answer_count": 1, "score": 2, "last_activity_date": 1597447025, "creation_date": 1596799465, "last_edit_date": 1597447025, "question_id": 63300819, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/63300819/adding-additional-loss-with-constant-zero-output-changes-model-convergence", "title": "Adding additional loss with constant zero output changes model convergence"}, {"tags": ["pytorch", "conv-neural-network"], "owner": {"account_id": 11910929, "reputation": 197, "user_id": 8716208, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/b3272d855884d926e8cc81c472711bfe?s=256&d=identicon&r=PG&f=1", "display_name": "DLH", "link": "https://stackoverflow.com/users/8716208/dlh"}, "is_answered": true, "view_count": 44, "accepted_answer_id": 73240325, "answer_count": 1, "score": 0, "last_activity_date": 1659635519, "creation_date": 1659628982, "question_id": 73239018, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/73239018/pytorch-fully-connected-layer-has-no-parameters", "title": "PyTorch: Fully Connected Layer has no Parameters"}, {"tags": ["pytorch-lightning"], "owner": {"account_id": 11203467, "reputation": 436, "user_id": 8219760, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/8cf09fae2d0a2d9c7a0cb5218305efeb?s=256&d=identicon&r=PG&f=1", "display_name": "vahvero", "link": "https://stackoverflow.com/users/8219760/vahvero"}, "is_answered": true, "view_count": 1612, "accepted_answer_id": 70674986, "answer_count": 1, "score": 1, "last_activity_date": 1641981528, "creation_date": 1641568665, "question_id": 70623377, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/70623377/running-test-calculations-in-ddp-mode-with-multiple-gpus-with-pytorchlightning", "title": "Running test calculations in DDP mode with multiple GPUs with PyTorchLightning"}, {"tags": ["deep-learning", "pytorch", "conv-neural-network", "kaggle"], "owner": {"account_id": 19225207, "reputation": 135, "user_id": 14048235, "user_type": "registered", "profile_image": "https://i.stack.imgur.com/Jp6q0.jpg?s=256&g=1", "display_name": "\u0410\u043d\u0434\u0440\u0435\u0439 \u0411\u0435\u0441\u043f\u0430\u043b\u043e\u0432", "link": "https://stackoverflow.com/users/14048235/%d0%90%d0%bd%d0%b4%d1%80%d0%b5%d0%b9-%d0%91%d0%b5%d1%81%d0%bf%d0%b0%d0%bb%d0%be%d0%b2"}, "is_answered": true, "view_count": 32, "accepted_answer_id": 73726731, "answer_count": 2, "score": -2, "last_activity_date": 1663224784, "creation_date": 1662890267, "question_id": 73678326, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/73678326/why-my-neural-network-isnt-learning-and-why-it-prediction-is-equal-on-all-test", "title": "Why my neural network isn`t learning and why it prediction is equal on all test dataset?"}, {"tags": ["python-3.x", "types", "pytorch"], "owner": {"account_id": 10576553, "reputation": 15, "user_id": 7791712, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/5d61ed3f3434bcf73d8cd34ccc63fad1?s=256&d=identicon&r=PG&f=1", "display_name": "Dheeraj", "link": "https://stackoverflow.com/users/7791712/dheeraj"}, "is_answered": true, "view_count": 1383, "accepted_answer_id": 59272827, "answer_count": 2, "score": 1, "last_activity_date": 1577603422, "creation_date": 1575999882, "question_id": 59272784, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/59272784/type-mismatch-in-pytorch", "title": "Type mismatch in pytorch"}, {"tags": ["neural-network", "recurrent-neural-network", "pytorch", "rnn"], "owner": {"account_id": 10032122, "reputation": 581, "user_id": 7418216, "user_type": "registered", "accept_rate": 100, "profile_image": "https://www.gravatar.com/avatar/3a5bc2097df700514804bb12a279b532?s=256&d=identicon&r=PG&f=1", "display_name": "Mahsa", "link": "https://stackoverflow.com/users/7418216/mahsa"}, "is_answered": true, "view_count": 1241, "accepted_answer_id": 50149430, "answer_count": 1, "score": 0, "last_activity_date": 1525333528, "creation_date": 1525331813, "last_edit_date": 1525331953, "question_id": 50149049, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/50149049/training-a-rnn-in-pytorch", "title": "training a RNN in Pytorch"}, {"tags": ["python-3.x", "segmentation-fault", "pytorch", "huggingface-transformers"], "owner": {"account_id": 8584097, "reputation": 1414, "user_id": 6430839, "user_type": "registered", "accept_rate": 100, "profile_image": "https://i.stack.imgur.com/YTZIH.jpg?s=256&g=1", "display_name": "Dr. Fabien Tarrade", "link": "https://stackoverflow.com/users/6430839/dr-fabien-tarrade"}, "is_answered": true, "view_count": 365, "accepted_answer_id": 65934307, "answer_count": 1, "score": 1, "last_activity_date": 1611825937, "creation_date": 1609863054, "question_id": 65582498, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/65582498/torch-nn-crossentropyloss-ignore-index-is-crashing-when-importing-transfomers", "title": "torch.nn.CrossEntropyLoss().ignore_index is crashing when importing transfomers library"}, {"tags": ["python", "pytorch"], "owner": {"account_id": 10536458, "reputation": 191, "user_id": 8307869, "user_type": "registered", "accept_rate": 100, "profile_image": "https://graph.facebook.com/1059299404214844/picture?type=large", "display_name": "JoZ", "link": "https://stackoverflow.com/users/8307869/joz"}, "is_answered": true, "view_count": 54, "accepted_answer_id": 71947131, "answer_count": 1, "score": 1, "last_activity_date": 1650496628, "creation_date": 1650495317, "question_id": 71947002, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/71947002/error-dimension-out-of-range-expected-to-be-in-range-of-1-0-but-got-1-wh", "title": "error: Dimension out of range (expected to be in range of [-1, 0], but got 1) when training to train a CNN model"}, {"tags": ["python", "machine-learning", "neural-network", "pytorch", "regression"], "owner": {"account_id": 21523725, "reputation": 398, "user_id": 15864414, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/f2833ab9521869de79e0beffc626d7ca?s=256&d=identicon&r=PG&f=1", "display_name": "Opps_0", "link": "https://stackoverflow.com/users/15864414/opps-0"}, "is_answered": true, "view_count": 84, "accepted_answer_id": 69185048, "answer_count": 1, "score": 0, "last_activity_date": 1631736980, "creation_date": 1631655219, "last_edit_date": 1631736132, "question_id": 69184746, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/69184746/how-to-get-continuous-value-from-output-layer", "title": "How to get continuous value from output layer"}, {"tags": ["python", "pytorch", "classification", "vgg-net"], "owner": {"account_id": 16170365, "reputation": 23, "user_id": 11674282, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/db04f73fa5e9bb90987d956b8418d17a?s=256&d=identicon&r=PG&f=1", "display_name": "sharktooth", "link": "https://stackoverflow.com/users/11674282/sharktooth"}, "is_answered": true, "view_count": 977, "accepted_answer_id": 72286912, "answer_count": 2, "score": 0, "last_activity_date": 1652867571, "creation_date": 1647691869, "question_id": 71538030, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/71538030/weight-tensor-should-be-defined-either-for-all-1000-classes-or-no-classes-but-go", "title": "Weight tensor should be defined either for all 1000 classes or no classes but got weight tensor of shape: [5]"}, {"tags": ["tensorflow", "keras", "pytorch"], "owner": {"account_id": 14064207, "reputation": 487, "user_id": 10159479, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/835dda9b64c9a803249e71b188548049?s=256&d=identicon&r=PG&f=1", "display_name": "Greg", "link": "https://stackoverflow.com/users/10159479/greg"}, "is_answered": true, "view_count": 1252, "answer_count": 1, "score": 3, "last_activity_date": 1563407489, "creation_date": 1563350013, "last_edit_date": 1563384654, "question_id": 57071035, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/57071035/compare-the-example-of-pytorch-and-keras-on-cifar10-data", "title": "Compare the example of Pytorch and Keras on Cifar10 data"}, {"tags": ["python", "bert-language-model", "allennlp"], "owner": {"account_id": 17006553, "reputation": 13, "user_id": 12302635, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/bb5b4cce259bc931281711143472058d?s=256&d=identicon&r=PG&f=1", "display_name": "gorkem", "link": "https://stackoverflow.com/users/12302635/gorkem"}, "is_answered": true, "view_count": 77, "accepted_answer_id": 66243368, "answer_count": 1, "score": 0, "last_activity_date": 1613570155, "creation_date": 1613568443, "question_id": 66242860, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/66242860/cannot-register-text-classifier-as-model-name-already-in-use-for-textclassifier", "title": "Cannot register text_classifier as Model; name already in use for TextClassifier"}, {"tags": ["python", "deep-learning", "neural-network", "pytorch", "learning-rate"], "owner": {"account_id": 22672022, "reputation": 143, "user_id": 16843630, "user_type": "registered", "profile_image": "https://i.stack.imgur.com/tq03R.png?s=256&g=1", "display_name": "Snehangsu", "link": "https://stackoverflow.com/users/16843630/snehangsu"}, "is_answered": false, "view_count": 266, "answer_count": 1, "score": 0, "last_activity_date": 1651979367, "creation_date": 1651817294, "last_edit_date": 1651827976, "question_id": 72136870, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/72136870/how-to-get-the-actual-learning-rate-in-pytorch", "title": "How to get the actual learning rate in PyTorch?"}, {"tags": ["python-3.x", "deep-learning", "recurrent-neural-network", "pytorch", "rnn"], "owner": {"account_id": 1298864, "reputation": 1778, "user_id": 1249184, "user_type": "registered", "accept_rate": 62, "profile_image": "https://i.stack.imgur.com/UkPsU.jpg?s=256&g=1", "display_name": "DukeLover", "link": "https://stackoverflow.com/users/1249184/dukelover"}, "is_answered": true, "view_count": 629, "answer_count": 1, "score": 0, "last_activity_date": 1524558949, "creation_date": 1524508046, "question_id": 49987673, "content_license": "CC BY-SA 3.0", "link": "https://stackoverflow.com/questions/49987673/how-can-i-build-an-rnn-without-using-nn-rnn", "title": "How can I build an RNN without using nn.RNN"}, {"tags": ["python", "nlp", "pytorch", "bert-language-model"], "owner": {"account_id": 11889698, "reputation": 1076, "user_id": 8700970, "user_type": "registered", "accept_rate": 40, "profile_image": "https://www.gravatar.com/avatar/729df62d3308d352b346d43b3326666a?s=256&d=identicon&r=PG&f=1", "display_name": "joel", "link": "https://stackoverflow.com/users/8700970/joel"}, "is_answered": false, "view_count": 367, "answer_count": 1, "score": 0, "last_activity_date": 1598566807, "creation_date": 1598563687, "question_id": 63624390, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/63624390/bert-binary-text-classifier-giving-valueerror-expected-input-batch-size-doesnt", "title": "BERT Binary Text Classifier giving ValueError: Expected input batch_size doesn`t match target"}, {"tags": ["python", "deep-learning", "neural-network", "pytorch", "conv-neural-network"], "owner": {"account_id": 14880573, "reputation": 54, "user_id": 10745369, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/d87ce3dca763ac306ab61b4a7887a552?s=256&d=identicon&r=PG&f=1", "display_name": "Janosch", "link": "https://stackoverflow.com/users/10745369/janosch"}, "is_answered": false, "view_count": 548, "answer_count": 1, "score": 0, "last_activity_date": 1624516918, "creation_date": 1624471565, "last_edit_date": 1624472338, "question_id": 68105079, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/68105079/pytorch-error-while-building-cnn-1only-batches-of-spatial-targets-supported-3", "title": "PyTorch Error while building CNN: &quot;1only batches of spatial targets supported (3D tensors) but got targets of size: : [1, 2, 64, 64]&quot;"}, {"tags": ["python", "machine-learning", "deep-learning", "computer-vision", "pytorch"], "owner": {"account_id": 6932879, "reputation": 177, "user_id": 5321842, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/4b51a43b5b8e1344541c9432b1ef032a?s=256&d=identicon&r=PG&f=1", "display_name": "krustybek", "link": "https://stackoverflow.com/users/5321842/krustybek"}, "is_answered": true, "view_count": 6358, "answer_count": 1, "score": 3, "last_activity_date": 1543249476, "creation_date": 1543229445, "question_id": 53479523, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/53479523/increasingly-large-positive-wgan-gp-loss", "title": "Increasingly large, positive WGAN-GP loss"}, {"tags": ["python", "machine-learning", "pytorch", "multiclass-classification"], "owner": {"account_id": 16706151, "reputation": 145, "user_id": 12074622, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/a78d40bd4b12aa7821a3b3561f054377?s=256&d=identicon&r=PG&f=1", "display_name": "MenNotAtWork", "link": "https://stackoverflow.com/users/12074622/mennotatwork"}, "is_answered": false, "view_count": 105, "answer_count": 1, "score": 2, "last_activity_date": 1636963262, "creation_date": 1636856207, "last_edit_date": 1636856520, "question_id": 69959778, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/69959778/multilayer-perceptron-for-multiclass-classification-task", "title": "Multilayer Perceptron for multiclass classification task"}, {"tags": ["python", "pytorch"], "owner": {"account_id": 20373256, "reputation": 21, "user_id": 14945024, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/95a0d7e8adb94de7426bed45f6e54f61?s=256&d=identicon&r=PG&f=1", "display_name": "Misha", "link": "https://stackoverflow.com/users/14945024/misha"}, "is_answered": true, "view_count": 1746, "answer_count": 2, "score": 2, "last_activity_date": 1651675045, "creation_date": 1622713237, "last_edit_date": 1622713560, "question_id": 67819077, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/67819077/runtime-error-cuda-out-of-memory-by-the-end-of-training-and-doesn-t-save-model", "title": "Runtime error: CUDA out of memory by the end of training and doesn\u2019t save model; pytorch"}, {"tags": ["python", "nlp", "pytorch", "recurrent-neural-network", "gated-recurrent-unit"], "owner": {"account_id": 302279, "reputation": 107937, "user_id": 610569, "user_type": "registered", "accept_rate": 90, "profile_image": "https://www.gravatar.com/avatar/0e9087f2672b0e4f28d91266acf9ce57?s=256&d=identicon&r=PG", "display_name": "alvas", "link": "https://stackoverflow.com/users/610569/alvas"}, "is_answered": true, "view_count": 565, "answer_count": 2, "score": 0, "last_activity_date": 1552636632, "creation_date": 1552466832, "question_id": 55137631, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/55137631/gru-language-model-not-training-properly", "title": "GRU Language Model not Training Properly"}, {"tags": ["keras", "pytorch"], "owner": {"account_id": 7129723, "reputation": 1404, "user_id": 5451433, "user_type": "registered", "accept_rate": 81, "profile_image": "https://www.gravatar.com/avatar/537c39d6b60e039ff66890c0613227e5?s=256&d=identicon&r=PG&f=1", "display_name": "Robin1988", "link": "https://stackoverflow.com/users/5451433/robin1988"}, "is_answered": true, "view_count": 658, "answer_count": 1, "score": 2, "last_activity_date": 1594493607, "creation_date": 1565399051, "last_edit_date": 1565400356, "question_id": 57438562, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/57438562/pytorch-implement-the-same-model-in-pytorch-and-keras-but-got-different-results", "title": "Pytorch-Implement the same model in pytorch and keras but got different results"}, {"tags": ["python-3.x", "token", "huggingface-transformers", "named-entity-recognition"], "owner": {"account_id": 6258007, "reputation": 1127, "user_id": 9907733, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/10907e4712608ed07a3834df4adbe9ee?s=256&d=identicon&r=PG&f=1", "display_name": "MAC", "link": "https://stackoverflow.com/users/9907733/mac"}, "is_answered": true, "view_count": 24, "accepted_answer_id": 73372795, "answer_count": 1, "score": 0, "last_activity_date": 1660646989, "creation_date": 1660551105, "question_id": 73358347, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/73358347/how-to-resolve-the-error-name-name-label-if-label-in-features0-keys-e", "title": "How to resolve the error name:_name = &quot;label&quot; if &quot;label&quot; in features[0].keys() else &quot;labels&quot; in Hugging face NER"}, {"tags": ["python", "jupyter-notebook", "fast-ai"], "owner": {"account_id": 14116292, "reputation": 279, "user_id": 10197987, "user_type": "registered", "profile_image": "https://lh5.googleusercontent.com/-05NyDnZ5dGg/AAAAAAAAAAI/AAAAAAAAAAA/AAnnY7qNIUkiOE2CnzUdaG18_fLtNCTneg/mo/photo.jpg?sz=256", "display_name": "oceandye", "link": "https://stackoverflow.com/users/10197987/oceandye"}, "is_answered": false, "view_count": 122, "answer_count": 1, "score": 1, "last_activity_date": 1590765287, "creation_date": 1590765085, "question_id": 62089123, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/62089123/how-to-export-fast-ai-model-from-jupyter-notebook-hosted-on-external-servergoog", "title": "How to export fast.ai model from Jupyter Notebook hosted on external server(google cloud compute instance)?"}]